bpfassets/libbpf/src/kepler.bpf.c

```diff
@@ -62,13 +65,12 @@ static inline u64 get_on_cpu_time(u32 cur_pid, u32 prev_pid, u64 cur_ts)
         // timestamp later than the recorded off-CPU event, or vice versa.
         if (cur_ts > *prev_ts)
         {
-            cpu_time = (cur_ts - *prev_ts) / 1000000; /*milisecond*/
+            cpu_time = (cur_ts - *prev_ts) / 1000000 ; // convert to ms
```

Comment:
Since we are getting the CPU time from SW counter, do we still need to calculate the CPU time?
I think we can clean up the code!